src/openrct2-ui/windows/Ride.cpp

```diff
@@ -1060,7 +1061,7 @@ static void window_ride_draw_tab_vehicle(rct_drawpixelinfo* dpi, rct_window* w)
         }
 
         // For any suspended rides, move image higher in the vehicle tab on the rides window
-        if (RideTypeDescriptors[ride->type].HasFlag(RIDE_TYPE_FLAG_IS_SUSPENDED))
+        if (rtd.HasFlag(RIDE_TYPE_FLAG_IS_SUSPENDED))
```

Comment:
```
        if (const auto& rtd = ride->GetRideTypeDescriptor(); rtd.HasFlag(RIDE_TYPE_FLAG_IS_SUSPENDED))
```
or 
```
        if (ride->GetRideTypeDescriptor().HasFlag(RIDE_TYPE_FLAG_IS_SUSPENDED))
```

Reply:
Think most people decided that using init statement wasn't preferred.

Reply:
@tupaschoal I find both suggestions to have way worse readability.

Reply:
I don't find the second that bad, but your call...